boards/rfduino/Makefile.features

```diff
@@ -0,0 +1,2 @@
+FEATURES_PROVIDED += cpp
+FEATURES_PROVIDED += periph_uart periph_gpio periph_random periph_rtt periph_cpuid
```

Comment:
you don't implement all this features, so adapt this list to your implementation